Framework/Algorithms/src/MaskBins.cpp

```diff
@@ -119,27 +93,12 @@ void MaskBins::exec() {
       this->findIndices(X, startBin, endBin);
     }
 
-    const int numHists = static_cast<int>(inputWS->getNumberHistograms());
+    const int numHists = static_cast<int>(indexSet.size());
```

Comment:
static_cast not required. One of the constructors for `Progress` takes a `size_t`